types/header.go

```diff
@@ -122,27 +121,21 @@ func (h *Header) ValidateBasic() error {
 	return nil
 }
 
-// MakeCometBFTVote make a cometBFT consensus vote for the sequencer to commit
-// we have the sequencer signs cometBFT consensus vote for compatibility with cometBFT client
-func (h *Header) MakeCometBFTVote() []byte {
-	vote := cmtproto.Vote{
-		Type:   cmtproto.PrecommitType,
-		Height: int64(h.Height()), //nolint:gosec
-		Round:  0,
-		// Header hash = block hash in rollkit
-		BlockID: cmtproto.BlockID{
-			Hash:          cmbytes.HexBytes(h.Hash()),
-			PartSetHeader: cmtproto.PartSetHeader{},
-		},
-		Timestamp: h.Time(),
-		// proposerAddress = sequencer = validator
+// Vote returns a vote for the header.
+func (h *Header) Vote() ([]byte, error) {
```

Comment:
We should pass a validator address to the vote right? And maybe also a signature?

Reply:
will always be one, ill add a check

Reply:
ah no need to add a check since it is just one